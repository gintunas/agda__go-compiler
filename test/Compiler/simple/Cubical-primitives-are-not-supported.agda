{-# OPTIONS --cubical #-}

open import Agda.Primitive.Cubical
