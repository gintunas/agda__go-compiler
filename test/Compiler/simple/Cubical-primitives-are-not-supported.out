COMPILE_FAILED

ret > ExitFailure 42
out > Not implemented: primIMin
out >
